src/main/java/com/hotel/api/book/model/request/HotelBookRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.hotel.api.book.model.request;
```

Comment:
Please follow the following path 
model -> book -> request -> HotelBookRequest.java